lists/doh/adblock/adaway

```diff
@@ -2119,6 +2119,68 @@
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
This doesn't feel right, why do we end up with all these blank lines?

Reply:
Fixed in latest commit